his/docfx.json

```
@@ -35,11 +35,10 @@
     "overwrite": [],
     "externalReference": [],
     "globalMetadata": {
-      "feedback_system": "Standard",
-      "breadcrumb_path": "/his/breadcrumb/toc.json",
+      "breadcrumb_path": "/host-integration-server/breadcrumb/toc.json",
       "layout": "Conceptual",
       "titleSuffix": "Host Integration Server",
-      "uhfHeaderId": "MSDocsHeader-His",
+      "uhfHeaderId": "MSDocsHeader-His"
     },
     "fileMetadata": {
       "author": {"troubleshooting/*.md": "christopherhouser"},
```

Christopher Houser no longer works at Microsoft:

```suggestion
      "author": {"troubleshooting/*.md": "tsnook"},
```